src/css/override_components.scss

```diff
@@ -135,15 +135,20 @@ button,
   }
 } /* stylelint-enable */
 
+.usa-button-disabled {
```

Comment:
This class is used on the Restart app button for applications that are stopped.

![screen shot 2017-01-11 at 1 52 03 pm](https://cloud.githubusercontent.com/assets/1449852/21861958/3327adb4-d805-11e6-9306-84e5e2759131.png)